barretenberg/cpp/src/barretenberg/stdlib/honk_recursion/verifier/verifier.test.cpp

```diff
@@ -234,26 +257,29 @@ template <typename OuterFlavor> class HonkRecursiveVerifierTest : public testing
 };
 
 // Run the recursive verifier tests with conventional Ultra builder and Goblin builder
-using Flavors = testing::Types<UltraFlavor, GoblinUltraFlavor>;
+using Flavors = testing::Types<GoblinUltraRecursiveFlavor_<GoblinUltraCircuitBuilder>,
+                               GoblinUltraRecursiveFlavor_<UltraCircuitBuilder>,
+                               UltraRecursiveFlavor_<UltraCircuitBuilder>,
+                               UltraRecursiveFlavor_<GoblinUltraCircuitBuilder>>;
 
-TYPED_TEST_SUITE(HonkRecursiveVerifierTest, Flavors);
+TYPED_TEST_SUITE(RecursiveVerifierTest, Flavors);
 
-HEAVY_TYPED_TEST(HonkRecursiveVerifierTest, InnerCircuit)
+HEAVY_TYPED_TEST(RecursiveVerifierTest, InnerCircuit)
 {
     TestFixture::test_inner_circuit();
 }
 
-HEAVY_TYPED_TEST(HonkRecursiveVerifierTest, RecursiveVerificationKey)
+HEAVY_TYPED_TEST(RecursiveVerifierTest, RecursiveVerificationKey)
 {
     TestFixture::test_recursive_verification_key_creation();
 }
 
-HEAVY_TYPED_TEST(HonkRecursiveVerifierTest, DISABLED_SingleRecursiveVerification)
+HEAVY_TYPED_TEST(RecursiveVerifierTest, DISABLED_SingleRecursiveVerification)
```

Comment:
oooof yeah forgot about this. We should re-enable these. They should not have been disabled, especially not without an issue. I'm assuming you enabled them to check that all of this is working. I know they were passing as of recently

Reply:
yeah I checked them, didn't quite understand why they are disabled, would be more than happy to enable